docs/tutorials/experiment_cloud_service.rst

```diff
@@ -105,8 +105,8 @@ which we’ve made public by editing the ``Share level`` field:
 
 .. jupyter-execute::
 
-    from qiskit_experiments.database_service import DbExperimentDataV1 as DbExperimentData
-    load_exp = DbExperimentData.load("96b86d51-5200-4270-8ac1-ce4c20188ab9", provider.service("experiment"))
+    from qiskit_experiments.framework.experiment_data import ExperimentData
+    load_exp = ExperimentData.load("96b86d51-5200-4270-8ac1-ce4c20188ab9", provider.service("experiment"))
```

Comment:
I think the service here needs to be changed to the experiment service or it won't work.